test/distributed/org/apache/cassandra/distributed/test/accord/AccordHostReplacementTest.java

```diff
@@ -0,0 +1,104 @@
+
+import static accord.utils.Property.qt;
+import static org.apache.cassandra.distributed.shared.ClusterUtils.stopUnchecked;
+import static org.apache.cassandra.utils.AccordGenerators.fromQT;
+import static org.apache.cassandra.utils.CassandraGenerators.insertCqlAllColumns;
+
+public class AccordHostReplacementTest extends TestBaseImpl
+{
+    private static final Logger logger = LoggerFactory.getLogger(AccordHostReplacementTest.class);
+
+    @Test
+    public void hostReplace() throws IOException
+    {
+        // start 2 node cluster and bootstrap 3rd node
+        // do a host replacement of one of the nodes
+        Cluster.Builder clusterBuilder = Cluster.build(3)
+                                                .withConfig(c -> c.with(Feature.values())
+                                                                  .set("accord.shard_count", "1"));
+        TokenSupplier tokenRing = TokenSupplier.evenlyDistributedTokens(3, clusterBuilder.getTokenCount());
+        int nodeToReplace = 2;
+        clusterBuilder = clusterBuilder.withTokenSupplier((TokenSupplier) node -> tokenRing.tokens(node == 4 ? nodeToReplace : node));
+        try (Cluster cluster = clusterBuilder.start())
+        {
+            fixDistributedSchemas(cluster);
+            init(cluster);
+
+            qt().withSeed(5215087322357346205L).withExamples(1).check(rs -> {
+                TableMetadata metadata = fromQT(new TableMetadataBuilder()
+                                                .withKeyspaceName(KEYSPACE)
+                                                .withTableKinds(TableMetadata.Kind.REGULAR)
+                                                .withKnownMemtables()
+                                                //TODO (coverage): include "fast_path = 'keyspace'" override
+                                                .withTransactionalMode(TransactionalMode.full)
+                                                .build())
+                                         .next(rs);
+                maybeCreateUDTs(cluster, metadata);
+                cluster.schemaChange(metadata.toCqlString(false, false));
+
+                Gen<ByteBuffer[]> dataGen = fromQT(fromQT(new CassandraGenerators.DataGeneratorBuilder(metadata).build(ignore -> 10)).next(rs));
+                String insertStmt = wrapInTxn(insertCqlAllColumns(metadata));
+
+                for (int i = 0; i < 10; i++)
+                    cluster.coordinator(1).execute(insertStmt, ConsistencyLevel.ANY, (Object[]) dataGen.next(rs));
+
+                cluster.forEach(i -> i.nodetoolResult("repair", "--full", "--accord-only", KEYSPACE).asserts().success());
+
+                stopUnchecked(cluster.get(nodeToReplace));
+                ClusterUtils.replaceHostAndStart(cluster, cluster.get(nodeToReplace));
+
+                cluster.forEach(i -> {
+                    if (i.isShutdown()) return;
+                    i.nodetoolResult("repair", "--full", "--accord-only", KEYSPACE).asserts().success();
```

Comment:
I think I added the repairs because I found a bug with `--accord-only`, it isn't required for the test and could be removed... doesn't hurt though.

Reply:
> writing with ALL

this is Accord, we don't look at `ConsistencyLevel` =D